status.json

```diff
@@ -1,7 +1,7 @@
 {
-  "code": "en",
-  "language": "English",
-  "maintainers": ["schacon"],
+  "code": "ms",
```

Comment:
You don't need to update the status.json file. It is no longer used.

Reply:
Alright, noted. Will revert the change.